doc/faq.md

```diff
@@ -42,3 +42,119 @@ Second, edit the assessment `pl-exp101/courseInstance/Fa17/assessments/final/inf
 ```
 
 See [Access control](accessControl.md) for more details.
+
+## Common Errors
```

Comment:
Rather than having a separate section here, how about just listing everything as level-2 headings in the FAQ? I like the text about "Have an error? Add it in the editor" -- this could just be moved to the top of the FAQ?

Reply:
I think as the FAQ grows it would make sense to have distinct sections afforded by the level-2 headings. Take for instance a course FAQ, there are registration, grading, OH sections. If all sections were placed together, there would be an accessibility issue.

Reply:
So, the main difference is the questions are pulled more out from the side bar...

**New**

![screen shot 2018-10-27 at 12 43 48 pm](https://user-images.githubusercontent.com/833642/47607473-14d4dc80-d9e6-11e8-8287-dc761b6e903f.png)

**Old**

![screen shot 2018-10-27 at 12 44 27 pm](https://user-images.githubusercontent.com/833642/47607472-143c4600-d9e6-11e8-994e-8e0a2a2bc910.png)

-----

In addition, the head mast provides some context:

**New**

![screen shot 2018-10-27 at 12 45 18 pm](https://user-images.githubusercontent.com/833642/47607493-4c438900-d9e6-11e8-95a0-9d67768467ad.png)

**Old**

![screen shot 2018-10-27 at 12 45 36 pm](https://user-images.githubusercontent.com/833642/47607492-4baaf280-d9e6-11e8-9f78-a7e652f95faf.png)

Reply:
Yep, this is the main reason that I'd rather only use two levels of heading in our docs (where possible). Top level is the file, then we have level-2 headings within that (which get shown in the UI). If our FAQ gets too big in the future then I'd prefer to just split it into multiple files. So for now let's just go with:

```
# FAQ

## Single question

## Another common question
```

etc.

Reply:
@mwest1066 addressed. Feel free to merge.